### ISSUE TITLE
[Adaptive Cards- Web Render Test on Teams]: Screen Reader is conveying twice role as 'Link and Image' while navigating on the 'Security Compliance'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#413]
### Target Platforms

iOS

### SDK Version

7.13

### Application Name

Microsoft Teams

### Problem Description

**Test Environment:**
Device: iPhone 12
iOS Version: 18.5
Teams Application Version: 7.13
Screen Reader: Voice Over

**Pre-Requisite:**
Navigate through voice over only.

**Repro Steps:**

1. Open the application using double tap.
2. Perform right swipe and navigate to the 'AC Web Render' group and double tap to activate it.
3. Perform right swipe and navigate to the 'Viva Learning' section and double tap to activate it.
4. Perform right swipe and navigate under the viva learning section and observe the issue.

**Actual Result:**
While navigating on the 'Security Compliance', voice over is conveying twice role as link and image both. Here when the focus lands on it then screen reader is conveying as **'Security Compliant link image then complete URL double tap to activate it.'**
Also, it is announcing the entire URL information while navigating on the link.

**Expected Result:**
Voice over should not announce twice role information while navigating on the 'Security Compliant link' and when the focus lands on it then voice over should announce as 'Security compliance link double tap to activate it.'

**User Impact:**
It will impact the screen reader user as they will get confused if the twice role announce by the screen reader as they will assume whether the control is link or image.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment

### Screenshots

https://github.com/user-attachments/assets/e3f5f48f-878d-4110-bfdc-36aeb8ee8d58

### Card JSON

```JSON
NA
```
